hazelcast/src/main/java/com/hazelcast/map/impl/client/MapGetRequest.java

```diff
@@ -126,9 +126,6 @@ public String getDistributedObjectName() {
 
     @Override
     public String getMethodName() {
```

Comment:
Can this lead to security of backwards incompatibility problems? This method name is afaik exposed in the security mechanism, so by changing it, the security mechanism is influenced and this could lead to sneaky incompatibility issues.